react/week2/to-do-app/src/ToDo.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+
+function ToDo(props){
```

Comment:
If you want, you can destructure the props object here to make it a little easier to read what exactly is passed to the component:
```suggestion
function ToDo({ id, handleDelete }){
```

Reply:
Thank you, i have resolved this part of code